parboiled/src/test/scala/org/parboiled2/Base64ParsingSpec.scala

```diff
@@ -48,18 +52,28 @@ object Base64ParsingSpec extends TestSuite {
     "base64CustomBlock"
   )
 
+  def testParser(encoded: String) = new TestParser(encoded) with DynamicRuleHandler[TestParser, Array[Byte] :: HNil] {
+    type Result = String
+    def parser: TestParser                           = this
+    def ruleNotFound(ruleName: String): Result       = "n/a"
+    def success(result: Array[Byte] :: HNil): Result = new String(result.head, UTF8)
+    def parseError(error: ParseError): Result        = throw error
+    def failure(error: Throwable): Result            = throw error
+  }
+
   def test(ruleName: String, base64: Base64): Unit =
     (1 to 100).foreach { x =>
```

Comment:
This would benefit from being a property test, but the `UTestScalaCheck` machinery isn't in this project.  It could be made so.

Reply:
Yes, true.
A rewrite of (or more serious further work on) parboiled2 would certainly bring property testing into the project. Just for this once occasion here I don't think it's worth it though...

Reply:
There's already a `UTestScalaCheck` trait in parboiled-core to support `CharUtilsSpec`.  With a `compile->compile;test->test` scope in `dependsOn`, I think it could be gradually introduced throughout the project.